### ISSUE TITLE
AlertDialog should not inherit a global menu bar

### ISSUE THREAD

**Issue by tkuenneth** [JetBrains/compose-jb#198]
Currently `AlertDialog` inherits a menubar. A complete but short demo is below. I think the window for the alert should be made configurable, and the menubar should not be shown. Here is how the demo looks at the moment:

![image](https://user-images.githubusercontent.com/13284437/102620919-14a47780-413f-11eb-9cdc-303ef888b7f8.png)

Here the demo that produces this output:

```kotlin
package com.thomaskuenneth

import androidx.compose.desktop.AppManager
import androidx.compose.desktop.Window
import androidx.compose.material.AlertDialog
import androidx.compose.material.Button
import androidx.compose.material.Text
import androidx.compose.runtime.mutableStateOf
import androidx.compose.runtime.remember
import androidx.compose.ui.window.Menu
import androidx.compose.ui.window.MenuBar
import androidx.compose.ui.window.MenuItem

fun main() {
    AppManager.setMenu(MenuBar(
            Menu("File", MenuItem("Quit", onClick = {
                AppManager.exit()
            }))
    ))
    Window(title = "AlertDialogDemo") {
        val show = remember { mutableStateOf(false) }
        Button(onClick = {
            show.value = true
        }) {
            Text("Show AlertDialog")
        }
        if (show.value) {
            AlertDialog(onDismissRequest = {},
                    title = {
                        Text("A title")
                    },
                    text = {
                        Text("A text")
                    },
                    confirmButton = {
                        Button(onClick = {}) {
                            Text("A button")
                        }
                    })
        }
    }
}
```

**Comment by Rsedaikin** [comment]
Fixed in version `0.3.0-build138`.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6494">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
